### ISSUE TITLE
temporary access for @addaleax to test-rackspace-centos7-x64-1

### ISSUE THREAD

**Issue by Trott** [nodejs/build#1794]
I'd like to provide @addaleax with temporary access to test-rackspace-centos7-x64-1 so she can troubleshoot on https://github.com/nodejs/node/issues/22762. @nodejs/build

**Comment by rvagg** [comment]
sgtm @Trott, I'm assuming you can take care of this?

**Comment by mhdawson** [comment]
+1

**Comment by sam-github** [comment]
Ooph. This was 10 days ago. @addaleax do you still need access? I think I can arrange it if you do.

**Comment by Trott** [comment]
Oh, sorry, yes, I was waiting for a second approval, although I see now that a single approval from someone other than the person raising the issue is enough. So I dropped the ball. 

@addaleax Let me (and/or Sam) know if you still want access. We can add your keys and pull the machine out of the rotation while you're working on it. Sorry about not following through on this!

**Comment by addaleax** [comment]
Yeah, I mean, the issue is still unresolved, so access to the machine would be great.

**Comment by Trott** [comment]
> Yeah, I mean, the issue is still unresolved, so access to the machine would be great.

OK, I've enabled your access. I'll contact you privately for further details.

**Comment by Trott** [comment]
(I'll also leave this issue open until we've removed your access.)

**Comment by addaleax** [comment]
https://github.com/nodejs/node/pull/27705 should be the PR that addresses this. I most likely don’t need access here anymore, but it would be nice to confirm that the fix works on this machine, because it seemed to reproduce somewhat well here.

That being said, I currently cannot log into the machine, as SSH does not successfully connect. I have not run anything out of the ordinary while logged in, only the standard steps to build Node.js. I don’t quite know what happened and I’d be glad if somebody who can take control of the machine in some way (other than SSH, I assume) could take a look.

**Comment by Trott** [comment]
Not sure how to reboot or otherwise access this machine without SSH.

https://ci.nodejs.org/computer/test-rackspace-centos7-x64-1/log

```console
JNLP agent connected from 119.9.27.82/119.9.27.82
Remoting version: 3.29
This is a Unix agent
Agent successfully connected and online
ERROR: Connection terminated
java.nio.channels.ClosedChannelException
	at org.jenkinsci.remoting.protocol.impl.ChannelApplicationLayer.onReadClosed(ChannelApplicationLayer.java:209)
	at org.jenkinsci.remoting.protocol.ApplicationLayer.onRecvClosed(ApplicationLayer.java:222)
	at org.jenkinsci.remoting.protocol.ProtocolStack$Ptr.onRecvClosed(ProtocolStack.java:816)
	at org.jenkinsci.remoting.protocol.FilterLayer.onRecvClosed(FilterLayer.java:287)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.onRecvClosed(SSLEngineFilterLayer.java:181)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.switchToNoSecure(SSLEngineFilterLayer.java:283)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.processWrite(SSLEngineFilterLayer.java:503)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.processQueuedWrites(SSLEngineFilterLayer.java:248)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.doSend(SSLEngineFilterLayer.java:200)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.doCloseSend(SSLEngineFilterLayer.java:213)
	at org.jenkinsci.remoting.protocol.ProtocolStack$Ptr.doCloseSend(ProtocolStack.java:784)
	at org.jenkinsci.remoting.protocol.ApplicationLayer.doCloseWrite(ApplicationLayer.java:173)
	at org.jenkinsci.remoting.protocol.impl.ChannelApplicationLayer$ByteBufferCommandTransport.closeWrite(ChannelApplicationLayer.java:314)
	at hudson.remoting.Channel.close(Channel.java:1452)
	at hudson.remoting.Channel.close(Channel.java:1405)
	at hudson.slaves.SlaveComputer.closeChannel(SlaveComputer.java:832)
	at hudson.slaves.SlaveComputer.access$800(SlaveComputer.java:105)
	at hudson.slaves.SlaveComputer$3.run(SlaveComputer.java:741)
	at jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)
	at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by addaleax** [comment]
Ok, tested on the machine and removed my keys, and landing the fix for #22762 soon, you can take it back online :)

**Comment by Trott** [comment]
Confirmed keys have been removed and put it back online. Will leave this open for a little bit to make sure host runs tests as expected and then close this out. Thanks!
